File: task-03/Ruby/2/Sub-Task2.rb
def read_and_write_file(input_file, output_file)
    begin
      content = File.read(input_file)
      
      File.write(output_file, content)
      
      puts "Content successfully copied from #{input_file} to #{output_file}."
    rescue StandardError => e
      puts "An error occurred: #{e.message}"
    end
  end
  
  read_and_write_file('input.txt', 'output.txt')
  

File: task-03/Elixir/2/Sub-Task2.exs
defmodule FileCopy do
  def read_and_write_file(input_file, output_file) do
    case File.read(input_file) do
      {:ok, content} ->
        case File.write(output_file, content) do
          :ok ->
            IO.puts("Content successfully copied from #{input_file} to #{output_file}.")
          {:error, reason} ->
            IO.puts("Error writing file #{output_file}: #{reason}")
        end
      {:error, reason} ->
        IO.puts("Error reading file #{input_file}: #{reason}")
    end
  end
end

FileCopy.read_and_write_file("input.txt", "output.txt")
